core/src/main/scala/com/metamx/tranquility/druid/DruidBeamMaker.scala

```diff
@@ -48,15 +48,17 @@ class DruidBeamMaker[A](
   indexService: IndexService,
   emitter: ServiceEmitter,
   objectWriter: ObjectWriter[A],
-  druidObjectMapper: ObjectMapper
+  druidObjectMapper: ObjectMapper,
+  taskContext: Map[String, Any] = Map.empty
```

Comment:
Why do we need default value for `taskContext` when in `DruidBeams` we already have default value for `taskContext` variable?